### PR TITLE
Avoid some redirects and minor nitpicks, including in the changelog

### PR DESCRIPTION
**Pull request by plehegar**
Nothing like publishing a document to find additional nitpicks. This aligns with the changes done in
 https://www.w3.org/2018/Process-20180201/

### CODE REVIEW

**Comment by dwsinger** [comment]
There are some subtle changes of wording here too.  Maybe we should get a few pairs of eyes over this?

**Comment by plehegar** [comment]
I'll try to get a +1 from chaals since I suspect he wrote the changelog. If there is disagreement, we'll default to what was in the call for review to the AC.

**Comment by wseltzer** [comment]
I proposed most of the edits to the changelog, for clarity. For example, a Rec can't be declared "superseded as well as obsolete" -- any given Rec can only be one or the other at a given time.

**Comment by swickr** [comment]
These LGTM (modulo a possible '##' in revised line 3433).

Since Process 2017 added Obsolete, it's more precise that the changelog for 2018 cite just the addition of Superseded.

**Comment by vfournier17** [comment]
I have some comments on the changelog.  

*Section 5.2.3
Members can require a 60-day minimum between Call for Review of a Charter that continues work on an existing WD and the Call for Participation for the group.
>>I propose changing this to: "An AC member may request an additional 60 days to review any new or substantively modified working group charter, with the additional review period occurring between the Call for Review and the Call for Participation."
*Section 6.7.2
W3C can produce an Amended Recommendation to update a Recommendation, without adding new features, parallel to the way a Working Group produces an Edited Recommendation.
>>These processes are not parallel, because in the case of an Edited Recommendation, there is still a working group to edit the specification.  In the case of an Amended Recommendation, there is no working group.  I would change "parallel" to "similar," or just describe how the Amended Recommendation process works.
*Section 6.9
A Recommendation can be declared Superseded.
>>Should rescinded and obsoleted also be mentioned?  Otherwise someone might read this and think the other two categories no longer exist.
*Section 9
Definition of MoU and conditions for approval.
>>Change to "The definition of "MOU" and conditions for approval have been clarified."

**Comment by chaals** [comment]
In general, the changelog is a pointer for where to look, not a substitute for reading the sections and the diff, so I don't think the text is particularly critical. That said...

@vfournier17 suggested:

> >*Section 5.2.3
Members can require a 60-day minimum between Call for Review of a Charter that continues work on an existing WD and the Call for Participation for the group.

>I propose changing this to: "An AC member may request an additional 60 days to review any new or substantively modified working group charter, with the additional review period occurring between the Call for Review and the Call for Participation."

No, because they cannot request **an additional** 60 days, just that the minimum review be 60 days.

>> *Section 6.7.2
W3C can produce an Amended Recommendation to update a Recommendation, without adding new features, parallel to the way a Working Group produces an Edited Recommendation.

>These processes are not parallel, because in the case of an Edited Recommendation, there is still a working group to edit the specification.  In the case of an Amended Recommendation, there is no working group.  I would change "parallel" to "similar," or just describe how the Amended Recommendation process works.

agreed.

>>*Section 6.9
A Recommendation can be declared Superseded.

>Should rescinded and obsoleted also be mentioned?  Otherwise someone might read this and think the other two categories no longer exist.

I don't think so. We would have mentioned such a change, and in the relevant section they are still there.

**Comment by vfournier17** [comment]
@chaals The description  in the changelog should not be incorrect.  So if you have objections to my suggestions, let's consider changes to improve it rather than rejecting it.  Alternatively, we could remove the text completely, and just mention that the sections have been changed and their headings without including a description of the change.  I'd be ok with that.

**Comment by chaals** [comment]
@vfournier17 I agree that the text should not be incorrect - hence agreeing (wholeheartedly, but it's not clear how to explain that in github) with the change I think improved the factual correctness. Of the others, I **think** one makes the statement less accurate, so oppose it, and the other seems not about accuracy but editorial preference so it could go either way (and others have already suggested that) without me worrying too much.

**Comment by vfournier17** [comment]
Ok, how about this:

Section 5.3
"An AC member may request at least 60 days to review any new or substantively modified working group charter, with such review period occurring between the Call for Review and the Call for Participation."

**Comment by dwsinger** [comment]
I have a hard time getting excited about change logs, as long as they tell us roughly when a change was made, and alert the reader to read the appropriate sections. So anything you guys agree on will probably be fine by me.

**Comment by plehegar** [comment]
ok, https://www.w3.org/2018/Process-20180201/ is up, with the latest from #159 (including the changes from @vfournier17 approved by @chaals )

File: index.html [hunk 1]
```diff
@@ -3430,7 +3430,7 @@ <h2 id="changes">14 Changes</h2>
         <dt><a href="#revised-rec">Section 6.7.2</a></dt>
 	<dd>W3C can produce an <a href="#rec-amended">Amended Recommendation</a>
         to update a Recommendation, without adding new features, parallel to the way a Working Group produces an Edited Recommendation.</dd>
-        <dt><a href="##rec-rescind">Section 6.9</a></dt>
```
Comment (Kometo2) [inline index.html]: :1st_place_medal:  i will wait with hope!

**Comment by nrooney** [comment]
Thanks for this!

**Pull request merged**